shell/renderer/api/electron_api_web_frame.cc

```diff
@@ -258,17 +259,32 @@ void SetName(v8::Local<v8::Value> window, const std::string& name) {
       blink::WebString::FromUTF8(name));
 }
 
-void SetZoomLevel(v8::Local<v8::Value> window, double level) {
+void SetZoomLevel(gin_helper::ErrorThrower thrower,
+                  v8::Local<v8::Value> window,
+                  double level) {
   content::RenderFrame* render_frame = GetRenderFrame(window);
+  if (!render_frame) {
+    thrower.ThrowError(
+        "Render frame was torn down before script could be executed");
```

Comment:
Can we change `script` in all these places to the method name.  E.g.

```suggestion
        "Render frame was torn down before script could be webFrame.setZoomLevel");
```